build.sbt

```diff
@@ -93,6 +93,7 @@ lazy val elasticsearch = project
   .enablePlugins(AutomateHeaderPlugin)
   .settings(
     name := "akka-stream-alpakka-elasticsearch",
+    parallelExecution in ThisBuild := false,
```

Comment:
Maybe add a comment explaining why this was needed?

Reply:
Thanks, I fixed to disable only test execution and added comment.